doc/manual/manual.tex

```diff
@@ -7008,6 +7008,104 @@ \subsubsection{Using reconstructed surface velocities by GPlates}
+asthenosphere, and high viscosities in the lower mantle. To make sure that these viscosity jumps 
+do not lead to numerical problems in our computation (see Section~\ref{sec:sinker-with-averaging}), 
+we also use harmonic averaging of the material properties.
+\lstinputlisting[language=prmfile]{cookbooks/burnman/material_model.part.prm}
+As the reference profile has a depth dependent density and also contains data for the compressibility, 
+this material model supports compressible convection models. 
+
+For the adiabatic conditions and the gravity model, we also specify that we want to use the respective
+\texttt{ascii data} plugin, and provide the data directory in the same way as for the material 
+model. The gravity model automatically uses the same file as the adiabatic conditions model.
+\lstinputlisting[language=prmfile]{cookbooks/burnman/adiabatic_conditions.part.prm}
+\lstinputlisting[language=prmfile]{cookbooks/burnman/gravity_model.part.prm}
+
+To make use of the reference state we just imported from BurnMan, we choose a formulation of the 
+equations that employs a reference state and compressible convection, in this case the anelastic 
+liquid approximation. 
+\lstinputlisting[language=prmfile]{cookbooks/burnman/formulation.part.prm}
+This means that the reference profiles are used for all material properties in the model, except for
+the density in the buoyancy term (on the right-hand side of the momentum conservation equation). 
+In addition, the density derivative in the mass conservation equation is taken from the adiabatic 
+conditions, where it is computed as depth derivative of the provided reference density profile
+(see also Section~\ref{sec:combined_formulations}).
+Without these lines in the parameter file, \aspect{} will use its default implementation for 
+compressible models, ``isothermal compression'' (see Section~\ref{sec:combined_formulations}), 
+which uses the full density everywhere in the equations, and uses the compressibility 
+to compute the density derivative term on the right-hand side of the mass conservation equation.
+In principle, the \texttt{ascii reference profile} material model also supports the truncated 
+anelastic liquid approximation (see Section~\ref{sec:tala}), which can be used by setting 
+the formulation of the equations to \texttt{anelastic liquid approximation} and specifying
+\lstinputlisting[language=prmfile]{cookbooks/burnman/tala.part.prm}
+This formulation will neglect any density changes due to dynamic pressure. 
+
+\paragraph{Visualizing the model output.}
+If we look at the output of our model (for example in \texttt{paraview}), we can see how cold, highly 
+viscous slabs are subducted and hot plumes rise from the core-mantle boundary. The final time step of 
+the model is shown in Figure~\ref{fig:burnman-convection}, and the full model evolution can be found 
+at \url{https://youtu.be/FS-ru4N0p-c}.
```

Comment:
No. But it would be great to have access, I have some more videos, and also a few minor things for the website.